Add a security policy under /security (#1661)
scala/scala-lang: security.md

---
title: Scala Security Policy
layout: inner-page-no-masthead
permalink: /security/
includeTOC: false
---

## Receiving Security Announcements

Security announcements related to Scala are published to the ["Security Announcements" channel](https://users.scala-lang.org/c/security) on our discourse forum.

Messages to this channel can only be posted by administrators, so it is very low traffic.
To set up email notifications for new security announcements, read [this post](https://users.scala-lang.org/t/about-the-security-announcements-category).

## Reporting Vulnerabilities

> We strongly encourage reporting security issues in Scala to our private mailing list before disclosing them in public.

Reviewer: ```suggestion
We strongly encourage reporting security issues in Scala to us privately before disclosing them in public.
```

The email address for security related communication is `security@scala-lang.org`.
Messages are delivered to the Scala Security Team, which includes people from EPFL, the Scala Center, VirtusLab and Lightbend.

> We strive to acknowledge reports within 24 hours.

Author: Extend this, e.g. to 3 days?

Reviewer 2: More importantly: business days ;)
May 2 business days?

In case you don't receive a reply within a few days and would like to escalate, our advice is to ask for a contact person in a forum hosted by the Scala organization:
  - [Meta category on Discourse](https://users.scala-lang.org/c/meta)

>   - [`#admin` channel on Discord](https://discord.com/channels/632150470000902164/632628729029328947) ([invite link](https://discord.com/invite/scala) for joining)

Author: Other options:
  - include a few direct contacts
  - open a ticket on scala/scala or scala/scala3

On a public forum, chat or also on a ticket, random people could jump in and give bad (or even malicious) advice.

Reviewer 2: I think the current options are fine. This shouldn't happen anyway, since the mailing list should be enough.

Author: I guess spam filtering is the possible concern here

Reviewer: regardless, I agree with Seb, this is already a good list of options
